_includes/configuration/content-style.md

```diff
@@ -11,6 +11,18 @@ These styles are useful for previewing content as it appears outside the editor.
 ```js
 tinymce.init({
   selector: 'textarea',  // change this value according to your HTML
-  content_style: "div {margin: 10px; border: 5px solid red; padding: 3px}"
+  content_style: 'div {margin: 10px; border: 5px solid red; padding: 3px; }'
```

Comment:
There should probably be a space in front of "margin" as well?

Reply:
fixed